brainscore/submission/evaluation.py

```diff
@@ -82,70 +116,97 @@ def run_evaluation(config_file, work_dir, jenkins_id, db_secret, models=None,
                         ceiled = score.sel(aggregation='center').item(0)
                         error = score.sel(aggregation='error').item(0)
                     finished = datetime.datetime.now()
+                    layer_commitment = str(
+                        model.layer_model.region_layer_map) if submission_entry.model_type == 'BaseModel' else ''
                     result = {
                         'Model': model_id,
-                        'Benchmark': benchmark,
+                        'Benchmark': benchmark_name,
                         'raw_result': raw,
                         'ceiled_result': ceiled,
                         'error': error,
                         'finished_time': finished,
-                        'layer' : str(model.layer_model.region_layer_map)
+                        'comment': {'layers': layer_commitment}
```

Comment:
just to make sure I get this right now: this comment will only be in the csv sent to the submitter and not show up in the database? (either way, shouldn't the `{'layers': layer_commitment}` just be a string `f"layers: {layer_commitment}"`?)

Reply:
Yes it is only in the csv, I made it a string!